synchronizer/l1_rollup_info_consumer.go

```diff
@@ -92,7 +92,7 @@ func (l *l1RollupInfoConsumer) step() error {
 		}
 		if rollupInfo.ctrlIsValid {
 			err = l.processIncommingRollupControlData(rollupInfo.ctrl)
-			if err != nil && !errors.Is(err, errConsumerStoppedBecauseIsSynchronized) {
+			if err != nil && !errors.Is(err, errConsumerStoppedBecauseIsSynchronized) && !errors.Is(err, errConsumerStopped) {
 				log.Error("consumer: error processing package.ControlData. Error: ", err)
```

Comment:
In this case what happens?? would it retries to get the package in the next iteration?

Reply:
It returns to main Sync function and depending of the lastblock returned is going to reset the synchronization process. This if is just to log an error or not. If the consumer finish by a controled situation we dont log a error